grafana/data_source_team_test.go

```diff
@@ -0,0 +1,31 @@
+package grafana
+
+import (
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatasourceTeam(t *testing.T) {
+	CheckOSSTestsEnabled(t)
+
+	var team gapi.Team
+	checks := []resource.TestCheckFunc{
+		testAccTeamCheckExists("grafana_team.test", &team),
+		resource.TestCheckResourceAttr(
+			"data.grafana_team.from_name", "name", "test-team",
+		),
```

Comment:
```suggestion
		resource.TestCheckResourceAttr(
			"data.grafana_team.from_name", "name", "test-team",
		),
		resource.TestMatchResourceAttr("data.grafana_team.from_name", "id", idRegexp),
```

Check the id as well

Reply:
Ok thanks, made the changes. Let me know if there is anything else